### PR TITLE
Update FortiAuthenticator with fixes and enhancements

### PR DESCRIPTION
**Pull request by jasonlo82**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/content-docs/issues/743

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 5.5.0
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by jasonlo82** [comment]
@yuvalbenshalom  Could you help to review and merge? Please let me know if any issues. Thanks.

**Review by yuvalbenshalom — COMMENTED** [review]
hi @jasonlo82
Thank you for the your contribution. I added 2 documentation comments please tell me if you agree with them.
BTW, did you manage to test the new arguments against the product?

**Review by yuvalbenshalom — APPROVED** [review]

File: Packs/FortiAuthenticator/Integrations/FortiAuthenticator/FortiAuthenticator.yml [hunk 1]
```diff
@@ -37,12 +35,15 @@ script:
       - localusers
       - ldapusers
       required: true
+    - name: email
+      description: The user's email that defined in the User Information on FortiAuthenticator.
```
Comment (amshamah419) [inline Packs/FortiAuthenticator/Integrations/FortiAuthenticator/FortiAuthenticator.yml:39]:
```suggestion
      description: The user's email that is defined in the User Information on FortiAuthenticator.
```

File: Packs/FortiAuthenticator/Integrations/FortiAuthenticator/FortiAuthenticator.yml [hunk 2]
```diff
@@ -37,12 +35,15 @@ script:
...
+    - name: username
+      description: The username that defined in the User Information on FortiAuthenticator.
```
Comment (amshamah419) [inline Packs/FortiAuthenticator/Integrations/FortiAuthenticator/FortiAuthenticator.yml:41]:
```suggestion
      description: The username that is defined in the User Information on FortiAuthenticator.
```

File: Packs/FortiAuthenticator/Integrations/FortiAuthenticator/FortiAuthenticator.yml [hunk 3]
```diff
@@ -37,12 +35,15 @@ script:
...
+    - name: token_serial
+      description: The serial no. of assigned Token on FortiAuthenticator.
```
Comment (amshamah419) [inline Packs/FortiAuthenticator/Integrations/FortiAuthenticator/FortiAuthenticator.yml:43]:
```suggestion
      description: The serial no. of the assigned Token on FortiAuthenticator.
```

File: Packs/FortiAuthenticator/Integrations/FortiAuthenticator/FortiAuthenticator.yml [hunk 4]
```diff
@@ -59,6 +60,10 @@ script:
       - localusers
       - ldapusers
       required: true
+    - name: email
+      description: The user's email that defined in the User Information on FortiAuthenticator.
```
Comment (amshamah419) [inline Packs/FortiAuthenticator/Integrations/FortiAuthenticator/FortiAuthenticator.yml:64]:
```suggestion
      description: The user's email that is defined in the User Information on FortiAuthenticator.
```

File: Packs/FortiAuthenticator/Integrations/FortiAuthenticator/FortiAuthenticator.yml [hunk 5]
```diff
@@ -59,6 +60,10 @@ script:
...
+    - name: username
+      description: The username that defined in the User Information on FortiAuthenticator.
```
Comment (amshamah419) [inline Packs/FortiAuthenticator/Integrations/FortiAuthenticator/FortiAuthenticator.yml:66]:
```suggestion
      description: The username that is defined in the User Information on FortiAuthenticator.
```

File: Packs/FortiAuthenticator/Integrations/FortiAuthenticator/README.md [hunk 6]
```diff
@@ -44,17 +44,24 @@ After you successfully execute a command, a DBot message appears in the War Room
 #### Input
 | **Argument Name** | **Description** | **Required** |
 | --- | --- | --- |
-| email | The user's email that defined in the User Information on FortiAuthenticator | Required | 
 | user_type | The user type:  localusers (Local Users), ldapuser (Remote Users) | Required | 
+| email | The user's email that defined in the User Information on FortiAuthenticator | Optional |
```
Comment (amshamah419) [inline Packs/FortiAuthenticator/Integrations/FortiAuthenticator/README.md:48]:
```suggestion
| email | The user's email that is defined in the User Information on FortiAuthenticator | Optional | 
```

File: Packs/FortiAuthenticator/Integrations/FortiAuthenticator/README.md [hunk 7]
```diff
@@ -44,17 +44,24 @@ After you successfully execute a command, a DBot message appears in the War Room
 #### Input
 | **Argument Name** | **Description** | **Required** |
 | --- | --- | --- |
-| email | The user's email that defined in the User Information on FortiAuthenticator | Required | 
 | user_type | The user type:  localusers (Local Users), ldapuser (Remote Users) | Required | 
+| email | The user's email that defined in the User Information on FortiAuthenticator | Optional | 
+| username | The username that defined in the User Information on FortiAuthenticator | Optional |
```
Comment (amshamah419) [inline Packs/FortiAuthenticator/Integrations/FortiAuthenticator/README.md:49]:
```suggestion
| username | The username that is defined in the User Information on FortiAuthenticator | Optional | 
```

File: Packs/FortiAuthenticator/Integrations/FortiAuthenticator/README.md [hunk 8]
```diff
@@ -44,17 +44,24 @@ After you successfully execute a command, a DBot message appears in the War Room
 #### Input
 | **Argument Name** | **Description** | **Required** |
 | --- | --- | --- |
-| email | The user's email that defined in the User Information on FortiAuthenticator | Required | 
 | user_type | The user type:  localusers (Local Users), ldapuser (Remote Users) | Required | 
+| email | The user's email that defined in the User Information on FortiAuthenticator | Optional | 
+| username | The username that defined in the User Information on FortiAuthenticator | Optional | 
+| token_serial | The serial no. of assigned Token on FortiAuthenticator | Optional |
```
Comment (amshamah419) [inline Packs/FortiAuthenticator/Integrations/FortiAuthenticator/README.md:50]:
```suggestion
| token_serial | The serial no. of the assigned Token on FortiAuthenticator | Optional | 
```

File: Packs/FortiAuthenticator/Integrations/FortiAuthenticator/README.md [hunk 9]
```diff
@@ -44,17 +44,24 @@ After you successfully execute a command, a DBot message appears in the War Room
 #### Input
 | **Argument Name** | **Description** | **Required** |
 | --- | --- | --- |
-| email | The user's email that defined in the User Information on FortiAuthenticator | Required | 
 | user_type | The user type:  localusers (Local Users), ldapuser (Remote Users) | Required | 
+| email | The user's email that defined in the User Information on FortiAuthenticator | Optional | 
+| username | The username that defined in the User Information on FortiAuthenticator | Optional | 
+| token_serial | The serial no. of assigned Token on FortiAuthenticator | Optional | 
+- Note: You need either email, username or token_serial input in order to get it works.
```
Comment (amshamah419) [inline Packs/FortiAuthenticator/Integrations/FortiAuthenticator/README.md:51]:
```suggestion
- Note: You need either an email, username, or token_serial input in order for the command to work.
```

File: Packs/FortiAuthenticator/Integrations/FortiAuthenticator/README.md [hunk 10]
```diff
@@ -76,26 +87,31 @@ After you successfully execute a command, a DBot message appears in the War Room
...
+|---|---|---|---|---|---|---|
+| 7 | test_user | test_user@example.com | true | true | ftm | FTKMOB123456789A |
 
 ### fortiauthenticator-update-user
 #### Input
 | **Argument Name** | **Description** | **Required** |
 | --- | --- | --- |
-| email | The user's email that defined in the User Information on FortiAuthenticator | Required | 
 | user_type | The user type:  localusers (Local Users), ldapuser (Remote Users) | Required | 
+| email | The user's email that defined in the User Information on FortiAuthenticator | Optional |
```
Comment (amshamah419) [inline Packs/FortiAuthenticator/Integrations/FortiAuthenticator/README.md:99]:
```suggestion
| email | The user's email that is defined in the User Information on FortiAuthenticator | Optional | 
```

File: Packs/FortiAuthenticator/Integrations/FortiAuthenticator/README.md [hunk 11]
```diff
@@ -76,26 +87,31 @@ After you successfully execute a command, a DBot message appears in the War Room
...
+| 7 | test_user | test_user@example.com | true | true | ftm | FTKMOB123456789A |
 
 ### fortiauthenticator-update-user
 #### Input
 | **Argument Name** | **Description** | **Required** |
 | --- | --- | --- |
-| email | The user's email that defined in the User Information on FortiAuthenticator | Required | 
 | user_type | The user type:  localusers (Local Users), ldapuser (Remote Users) | Required | 
+| email | The user's email that defined in the User Information on FortiAuthenticator | Optional | 
+| username | The username that defined in the User Information on FortiAuthenticator | Optional |
```
Comment (amshamah419) [inline Packs/FortiAuthenticator/Integrations/FortiAuthenticator/README.md:100]:
```suggestion
| username | The username that is defined in the User Information on FortiAuthenticator | Optional | 
```

File: Packs/FortiAuthenticator/Integrations/FortiAuthenticator/README.md [hunk 12]
```diff
@@ -76,26 +87,31 @@ After you successfully execute a command, a DBot message appears in the War Room
...
 ### fortiauthenticator-update-user
 #### Input
 | **Argument Name** | **Description** | **Required** |
 | --- | --- | --- |
-| email | The user's email that defined in the User Information on FortiAuthenticator | Required | 
 | user_type | The user type:  localusers (Local Users), ldapuser (Remote Users) | Required | 
+| email | The user's email that defined in the User Information on FortiAuthenticator | Optional | 
+| username | The username that defined in the User Information on FortiAuthenticator | Optional | 
 | active | Define user's active status:  false = Disabled, true = enabled | Required | 
+- Note: You need either email or username input in order to get it works.
```
Comment (amshamah419) [inline Packs/FortiAuthenticator/Integrations/FortiAuthenticator/README.md:102]:
```suggestion
- Note: You need either an email or username input in order for the command to work.
```

File: Packs/FortiAuthenticator/ReleaseNotes/1_0_1.md [hunk 13]
```diff
@@ -0,0 +1,4 @@
+#### Integrations
+##### FortiAuthenticator
+- Fixed an issue where **fortiauthenticator-get-user** command failed on LDAP users search.
```
Comment (amshamah419) [inline Packs/FortiAuthenticator/ReleaseNotes/1_0_1.md:3]:
```suggestion
- Fixed an issue where the ***fortiauthenticator-get-user*** command failed on LDAP user searches.
```

File: Packs/FortiAuthenticator/ReleaseNotes/1_0_1.md [hunk 14]
```diff
@@ -0,0 +1,4 @@
...
+- Added the arguments **username* and **token_serial** to the following commands: **fortiauthenticator-update-user** and **fortiauthenticator-update-user**.
```
Comment (amshamah419) [inline Packs/FortiAuthenticator/ReleaseNotes/1_0_1.md:4]:
```suggestion
- Added the arguments **username* and **token_serial** to the following commands: ***fortiauthenticator-update-user*** and ***fortiauthenticator-update-user***.
```

**Comment by amshamah419** [comment]
@jasonlo82 @yuvalbenshalom Just some minor documentation changes. If you use the "Add to Batch" command you can add apply the suggested changes in one commit.

**Pull request merged**